src/containers/AddPost/AddPost.js

```diff
@@ -1,16 +1,47 @@
 import React, {Component} from 'react';
 import TextField from "@material-ui/core/TextField/TextField";
 import Button from "@material-ui/core/Button";
+import Alert from '@material-ui/lab/Alert';
 import Navigations from "../../components/Navigations/Navigations";
 import classes from './AddPost.module.css';
+import * as actions from '../../store/actions/index';
+import {connect} from 'react-redux';
+import AlertTitle from "@material-ui/lab/AlertTitle";
+import Spinner from "../../components/UI/Spinner";
+let alert = null;
 
 class AddPost extends Component {
+
+    state = {
+        title: null,
+        description: null
+    }
+
+    addPostHandler = () => {
+        if (this.state.title && this.state.description){
+            alert = <Alert severity="success"><AlertTitle>Success</AlertTitle>Your post added successfully!</Alert>
```

Comment:
JSX logic should be handled in the render function

Reply:
Maybe we can add a new state for the alert and manage showing/hiding alert based on that